src/assets/i18n/pt-PT.json5

```diff
@@ -215,8 +218,11 @@
   // "admin.registries.bitstream-formats.title": "Bitstream Format Registry",
   "admin.registries.bitstream-formats.title": "Registo de formatos de ficheiro",
 
-  // "admin.registries.bitstream-formats.select": "Select bitstream format",
-  "admin.registries.bitstream-formats.select": "Selecionar formato de ficheiro",
+  // "admin.registries.bitstream-formats.select": "Select",
+  "admin.registries.bitstream-formats.select": "Selecionar",
```

Comment:
Portuguese from Portugal is "Seleccionar" instead of "Selecionar" which is the Portuguese from Brasil.